infra/charts/turing/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
 description: "Turing: ML Experimentation System"
 name: turing
-version: 0.2.4-dev0.1
+version: 0.2.4-dev0.2
```

Comment:
General question: I see we are using the pre-release `-dev` labelling on the version now. What's our convention with this? In general, the chart versions would not be coupled to the app releases, right? How do we meaningfully track the chart version along with the app version (I guess this is something we only worry about for breaking changes to the chart)?

Reply:
There is no convention as of now. I started using this `-dev` suffix in https://github.com/gojek/turing/pull/153 to make a point, that this chart version is not intended to be a stable chart release (as it didn't support the deployment with plugins fully). Also, I'm publishing these dev charts into my personal GitHub repo for testing, before the PR is fully done, so I can test the deployment with plugins in the GKE environment.
I will update the helm chart with a stable version before merging this PR.